libpoolprotocols/PoolManager.cpp

```diff
@@ -86,6 +86,7 @@ PoolManager::PoolManager(boost::asio::io_service& io_service, PoolClient* client
         {
             cnote << "New epoch " EthWhite << wp.epoch << EthReset;
             m_lastEpoch = wp.epoch;
+            m_epochChanges.fetch_add(1, std::memory_order_relaxed);
```

Comment:
You can use m_epochChanges++ here. In C++11 the ++ operator is supported by the atomic template.